### ISSUE TITLE
Grad_norm is Nan when resume training

### ISSUE THREAD

**Issue by GeorgePearse** [BinItAI/visdet#988]
**Original issue:** https://github.com/open-mmlab/mmdetection/issues/7189
**Created:** 2022-02-17
**Last updated:** 2023-06-30

---

TorchVision: 0.10.1+cu111
OpenCV: 4.5.3
MMCV: 1.3.9
MMCV Compiler: GCC 7.5
MMCV CUDA Compiler: 11.1
MMDetection: 2.12.0+/2.14.0+
TORCH_VERSION=1.9.1

Train SparseRCNN_R50_8stages
Train on single GPU 3080ti. Each GPU has 4 images. Cumulate gradients for 4 iterations to backpropagate.
Nothing else has been changed compared to the "sparse_rcnn_r50_fpn_1x_coco.py"
When I resumed training, I found that grad_norm become NaN but the original training (training at one time) will not have any NaN value.

If the checkpoint file is needed, I am very happy to upload it.
